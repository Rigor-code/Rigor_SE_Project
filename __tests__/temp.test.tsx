import React from 'react';
import renderer from 'react-test-renderer';
import Greeting from '../app/(tabs)/greetings';

describe('Greeting component', () => {
  it('renders correctly with given name', () => {
    const tree = renderer.create(<Greeting name="Nomaan" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});