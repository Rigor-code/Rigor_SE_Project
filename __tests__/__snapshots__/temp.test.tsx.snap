// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Greeting component renders correctly with given name 1`] = `
<View>
  <Text>
    Hello, 
    Nomaan
    !
  </Text>
</View>
`;
